database.py

```diff
@@ -93,6 +93,14 @@ def get_extra(self):
         # Get all the Extra Pokemon images available.
         return self.__get_region(None)
 
+    def get_light(self):
```

Comment:
Light and dark themes would be cool.

```python
# one function that accepts arg instead of two
# do not .strip() all lines, only strip the chosen line
def get_light_or_dark(self, s='light'):
    with open(self.directory + "/./Data/" + s.lower() + ".txt") as in_file:
        return int(random.choice(in_file.readlines()).strip())

# and then in main.py...
if arg in ("light", "dark"):
    change_terminal_background(db, db.get_light_or_dark(arg))
```